homeassistant/components/microsoft/tts.py

```diff
@@ -82,6 +82,7 @@
         vol.Optional(CONF_LANG, default=DEFAULT_LANG): vol.In(SUPPORTED_LANGUAGES),
         vol.Optional(CONF_GENDER, default=DEFAULT_GENDER): vol.In(GENDERS),
         vol.Optional(CONF_TYPE, default=DEFAULT_TYPE): cv.string,
+        vol.Optional(CONF_OUTPUT, default=DEFAULT_OUTPUT): cv.string,
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Reply:
Yes I know... may we have an exception for this? I mean it is just 1 missing parameter (which should have been a parameter from the start, I don't know why it's the only one hard-coded). Developing a UI would be a much greater effort. Please don't make me use a patch for this tiny thing 😞 
Thank you

Reply:
ADR-0010 for the UI might be much for TTS at this point indeed in general. @MartinHjelmare Pointed out that TTS and config entries don't work together yet. This means ADR-0010 does not apply to this PR.

However, ADR-0007 does, which does need to be implemented first.

Reply:
> However, ADR-0007 does, which does need to be implemented first.

I'm sorry I don't think I can put effort into that at the moment :-(
I guess I'll use my patch on my istance for now.

Reply:
Alright, I guess in that case we close this PR until a later moment.

Nevertheless, thanks for willing to contribute 👍